components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/BraveSiteSettingsCategory.java

```diff
@@ -22,6 +24,8 @@ public static String preferenceKey(@Type int type) {
         switch (type) {
             case Type.AUTOPLAY:
                 return "autoplay";
+            case Type.GOOGLE_SIGN_IN:
+                return "google_sign_in";
```

Comment:
I believe this string should be in sync with the one in `ContentSettingsRegistry::BraveInit`.